test/test_tensor_creation_ops.py

```diff
@@ -1414,6 +1414,32 @@ def test_meshgrid_default_indexing(self, device):
         self.assertTrue(grid_b2.equal(expected_grid_b))
         self.assertTrue(grid_c2.equal(expected_grid_c))
 
+    def test_meshgrid_xy_indexing(self, device):
+        a = torch.tensor(1, device=device)
```

Comment:
This is a cool "golden values" test but we should look at creating or generating a lot of inputs and validating that PyTorch's and NumPy's meshgrid have the same behavior, too

Reply:
In addition to using `numpy.meshgrid` as `OpInfo.reference`? How should these tests differ? Which should be more extensive?

Reply:
The SampleInputs should be relatively small because they're used to generate so many tests

Reply:
implemented in parent PR and extended here to do "xy" comparison.